app/views/spree/components/forms/inputs/_text.html.erb

```diff
@@ -2,11 +2,14 @@
   base_class = "text-input".freeze
 
   # Optional props
+  autocomplete = local_assigns.fetch(:autocomplete, false)
+  autofocus = local_assigns.fetch(:autofocus, false)
   id = local_assigns.fetch(:id, false)
   disabled = local_assigns.fetch(:disabled, false)
   label = local_assigns.fetch(:label, false)
   min = local_assigns.fetch(:min, false)
-  placeholder = local_assigns.fetch(:placeholder, '')
+  placeholder = local_assigns.fetch(:placeholder, false)
```

Comment:
Why this changed from `''` to `false`?

Reply:
Because otherwise the placeholder attribute is rendered with an empty string in the HTML. Ideally we don't want it at all if we don't pass it to the component.